src/AzureIoTHub.Portal/Server/Controllers/v1.0/DashboardController.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace AzureIoTHub.Portal.Server.Controllers.v1._0
+{
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Shared.Models.v1._0;
+
+    [Authorize]
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Route("api/dashboard")]
+    [ApiExplorerSettings(GroupName = "IoT Devices")]
```

Comment:
Change the Group name to ``Metrics``
```suggestion
    [ApiExplorerSettings(GroupName = "Metrics")]
```